src/pool.py

```diff
@@ -0,0 +1,18 @@
+from typing import  TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from .client import DiscordClient
+
+class pool:
+    
+    def __init__(self, tokens: list[str]) -> None:
+        self.tokens: list[str] = tokens
+        self.cursors: list["DiscordClient"] = [DiscordClient.login(token) for token in tokens]
```

Comment:
```suggestion
        self.cursors: list["DiscordClient"] = [DiscordClient.login(token) for token in tokens]
```
Because `DiscordClient` is only defined when `TYPE_CHECKING` is True (during type checking), this will error in runtine

Reply:
fixed in newer commit